Lib/fontTools/pens/decomposingPen.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function, division, absolute_import
+from fontTools.misc.py23 import *
+from fontTools.pens.filterPen import FilterPen
+from fontTools.pens.transformPen import TransformPen
+
+
+class DecomposingPen(FilterPen):
+
+    """A filter pen that decomposes components before drawing them with
+    another pen.
+    """
+
+    def __init__(self, outPen, glyphSet):
+        super(DecomposingPen, self).__init__(outPen)
```

Comment:
> In this case, it is assumed that FilterPen is called, ie. the call arguments are that of FilterPen, not any arbitrary parent.

True. But one may also wish to call the `__init__` of a subcass of FilterPen, no?